### PR TITLE
[chrony] Sync time on Nodes with control plane.

### PR DESCRIPTION
**Pull request by mvasl**
## Description

`470-chrony` is now deployed to all nodes at once, but master nodes are configured run 2 chrony processes at once, one acts as an intermediate NTP server, pulling clock from some other NTP service if possible, other works as regular chrony NTP client. 

Chrony clients are configured to use master nodes as their primary NTP clock source (`stratum 5`) via a headless service acting as an NTP pool, and fall back to NTP servers from module config in case something bad happens with masters.

## Why do we need it, and what problem does it solve?

See #2949

## Checklist
- [x] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: chrony
type: feature
summary: Master nodes act as NTP servers for cluster.
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: ee/modules/030-cloud-provider-openstack/candi [hunk 1]
```diff
@@ -1 +1 @@
-/deckhouse/candi/cloud-providers/openstack/
\ No newline at end of file
+/deckhouse/candi/cloud-providers/openstack
```
Comment (RomanenkoDenys) [inline ee/modules/030-cloud-provider-openstack/candi:1]: remove change symlink from PR please

File: ee/modules/030-cloud-provider-vsphere/candi [hunk 2]
```diff
@@ -1 +1 @@
-/deckhouse/candi/cloud-providers/vsphere/
\ No newline at end of file
+/deckhouse/candi/cloud-providers/vsphere
```
Comment (RomanenkoDenys) [inline ee/modules/030-cloud-provider-vsphere/candi:1]: remove change symlink from PR please

File: helm_lib/charts/deckhouse_lib_helm/templates/_node_affinity.tpl [hunk 3]
```diff
@@ -62,7 +62,7 @@ nodeSelector:
 {{- /* Usage: {{ include "helm_lib_tolerations" (tuple . "any-node" "with-uninitialized" "without-storage-problems") }} */ -}}
 {{- define "helm_lib_tolerations" }}
   {{- $context := index . 0 }}  {{- /* Template context with .Values, .Chart, etc */ -}}
-  {{- $strategy := index . 1 | include "helm_lib_internal_check_tolerations_strategy" }} {{- /* base strategy, one of "frontend" "monitoring" "system" any-node" "wildcard" */ -}}
+  {{- $strategy := index . 1 | include "helm_lib_internal_check_tolerations_strategy" }} {{- /* base strategy, one of "frontend" "monitoring" "system" any-node" "wildcard" "only-masters" */ -}}
```
Comment (RomanenkoDenys) [inline helm_lib/charts/deckhouse_lib_helm/templates/_node_affinity.tpl:65]: We shouldn't change base strategies

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Review by name212 — APPROVED** [review]

**Pull request merged**